index.html

```diff
@@ -318,21 +284,6 @@ <h2 class="contact-me">Contact me</h2>
     </form>
   </fieldset>
 </div>
-=======
-<h2 class="contact-me">Contact me</h2>
-<p class="info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-
-<fieldset class="fieldset">
-  <form action="https://formspree.io/f/xbjbneev" method="post">
-    <ul>
-    <li><input class="input-group" type="text" maxlength="30" name="name" placeholder="Your Name" required></li>
-    <li><input class="input-group" type="email" name="email" placeholder="Your Email" required></li>
-    <li><textarea class="input-group text-area" name="user_message" maxlength="500" placeholder="Write your message here" required></textarea></li>
-    <button class="form-button" type="submit">Get in Touch</button>
-    </ul>
-  </form>
```

Comment:
- [ ] Good job Implemeting the changes, I think while making the changes another issue is introduced, the textarea and the submit button which is get in touch is not appearing in the viewport and is going outside of the viewport area, I am attaching a scrrenshot of the whole viewport please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/208252316-09582458-b15f-4063-a56e-5f87d3d6f246.png)

Please fix this issue as it is one of the Important part of our portfolio and we have to make sure that the user is able to submit the form and contact us